CONTRIBUTING.md

```diff
@@ -8,7 +8,10 @@ There are a few basic ground-rules for contributors (including the maintainer(s)
 - **Non-master branches**, prefixed with a short name moniker (e.g. `gav-my-feature`) must be used for ongoing work.
 - **All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
 - A pull-request _must not be merged until CI_ has finished successfully.
-- Contributors should adhere to the [house coding style](https://github.com/paritytech/polkadot/wiki/Style-Guide).
+- Contributors should apply `cargo +nightly fmt` with the version used in CI.
```

Comment:
```suggestion
- Contributors should apply `cargo +nightly fmt` with the version used in CI (up-to-date CI image: `paritytech/ci-linux:production`).
```

Reply:
I was under the impression we could use the `rust-toolchain` file for specifying the the supported toolchain rather than pulling in the whole container env, but it seems it only supports a single rust version reference and being unspecific regarding the tool https://github.com/rust-lang/rustup/pull/1172